app/views/shared/_search_context_sidebar.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="panel panel-default">
+  <h3 class="sr-only panel-title">Search context</h3>
+  <div class="panel-body">
+    <h3><%= t('blacklight.search.zero_results.search_context') %></h3>
+    <div class="media">
+      <div class="media-body">
+        <%= link_to_catalog_search unless controller_name == 'catalog'%>
+        <%= link_to_article_search unless article_search? %>
+        <%= link_to_library_website_search %>
+        <%# TODO: add bento search link%>
```

Comment:
Is this still applicable?

Reply:
✅ added an issue number (#1695) to the comment. This will be resolved by the end of the work cycle.